### ISSUE TITLE
[Live-MAS Violation]: When pressed Enter key on the “Back to Top” link, and navigating using the tab key, Screen reader is not narrating any information when the focus is on “Buy From Microsoft”.

### ISSUE THREAD

**Issue by ChristopherSmithMicrosoft** [nvaccess/nvda#9952]
User Experience: Users who rely on screen readers will be affected if When pressed Enter key on the “Back to Top” link, and navigating using the tab key, Screen reader is not narrating any information when the focus is on “Buy From Microsoft”.

Test Environment:
OS: Windows 10
Browser: Chrome
URL:  www.microsoft.com/en-us/sql-server/sql-server-2017-editions
Screen readers: NVDA

Repro Steps:
Step 1: Open the above URL using Chrome Browser.
Step 2: Navigate to the “Back to top” link present in the page and press Enter key on it.
Step 3: On next tab key press, verify if the screen reader is narrating any information of “Buy from Microsoft” link or not.

Actual:

When pressed Enter key on the “Back to Top” link, and navigating using the tab key, Screen reader is not narrating any information when the focus is on “Buy From Microsoft”.

Expected:

When pressed Enter key on the “Back to Top” link, and navigating using the tab key, Screen reader should narrate relavant information when the focus is on “Buy From Microsoft”.

MAS Reference:
MAS 4.1.2
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Comment by josephsl** [comment]
Hi, just to make sure: how about on new Edge powered by Chromium? CC @JCSTeh

**Comment by josephsl** [comment]
Hi, update: looks like this is unique to Chromium-based browsers (I can reproduce this on Edge Dev 77). One workaround is to press Tab once more and press Shift+Tab. Thanks.

**Comment by josephsl** [comment]
Hi, next follow-up: is this reproducible when using Narrator in 19H1 onwards? If yes, I’m thinking this is something that may need to be resolved from Microsoft, otherwise it might be us. Thanks.

**Comment by ChristopherSmithMicrosoft** [comment]
Yes, we are experiencing the same issue on the new Edge browser as well, however when we use Edge and native Narrator screen reader it's working as expected.

**Comment by josephsl** [comment]
Hi, yep, I can confirm that (note that I’m using latest NVDA alpha build to test things). Let’s hear from web experts here so we can figure out what’s up. Thanks.

**Comment by Adriani90** [comment]
This works as expected in Edge  Cromoum and Google Crome 81 with NVDA Version: alpha-20015,56a62097. Closing as works for me.
Feel free to comment if you still have this issue with these versions and we can reopen it.
